components/preparing_workspace/preparing_workspace.tsx

```diff
@@ -242,94 +198,6 @@ export default function PreparingWorkspace(props: Props) {
 
     const sendForm = async () => {
         const sendFormStart = Date.now();
-        setSubmissionState(SubmissionStates.Submitting);
```

Comment:
We'll still want to set `SubmissionState.Submitting` so that any changes to props (for example because a saving of a preference changed some values) don't refire the useEffect sending of the form.

Reply:
ok, thank you!